Test.cs

```diff
@@ -4,6 +4,7 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
         // Method to calculate the factorial of a number
         public static long calculate(long number)
         {
```

Comment:
Method `calculate` should be renamed to `Calculate` to follow PascalCase naming convention.